### PR TITLE
kubevirtci: Introduce bump script

### PR DESCRIPTION
**Pull request by oshoval**
**What this PR does / why we need it**:
Built such that automation can use it:
In case fatal happens return error code.
In case all is good, determine if there was a change
according non empty stdout.

**Special notes for your reviewer**:

**Release note**:
```release-note
None
```

### CODE REVIEW

File: hack/bump-kubevirtci.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -e
+
+KUBEVIRTCI_TAG=$(curl -L -Ss https://storage.googleapis.com/kubevirt-prow/release/kubevirt/kubevirtci/latest)
+[[ ${#KUBEVIRTCI_TAG} != "18" ]] && echo "error getting KUBEVIRTCI_TAG" && exit 1
```
Comment (rhrazdil) [inline hack/bump-kubevirtci.sh:4]: Could you assign the magic number to a constant to make clear what is the purpose?
eg. EXPECTED_KUBEVIRTCI_TAG_LENGTH

**Comment by oshoval** [comment]
Thanks Radim
@RamLavi 
Can you take a look please ?

**Comment by oshoval** [comment]
> Also, do you plan to use it from other repos like project-infra?
we don't going to do automation at the moment if thats what you mean

**Comment by oshoval** [comment]
Added makefile target

**Review by RamLavi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**